deploy/kubernetes/elasticsearch/elasticsearch.gcloud.yaml

```diff
@@ -18,6 +18,10 @@ spec:
     tls:
       selfSignedCertificate:
         disabled: true
+  {% if ES_CONFIGURE_SNAPSHOTS %}
+  secureSettings:
+    - secretName: es-snapshot-gcs-secrets
```

Comment:
I have found that a lot of ECK's auto-magic is really heavily dependent on naming. The documentation names this secret `gcs-credentials`  but you've named it something else, and I don't see this secret being referenced anywhere else in your PR. Just confirming that this being used properly

Reply:
Yes, this should be correct. What matters isn't the name of the containing secret, but the name of the *data* inside the secret. In this case, we're ending up with a key inside of `es-snapshot-gcs-secrets` called `gcs.client.default.credentials_file`, which tells elasticsearch that it's the key for a GCS client called `default`. When configuring the repository, we set the name of the client: https://github.com/broadinstitute/seqr/pull/1663/files#diff-0c198e3b884d6600c5301859c8677080b8878c0582f4f03075733d3510e9939aR12

see https://www.elastic.co/guide/en/elasticsearch/plugins/master/repository-gcs-usage.html#repository-gcs-using-service-account

Reply:
also re: the secret not being mentioned anywhere else in the PR: `es-snapshot-gcs-secrets` is the name of the secret as generated by `deploy_secrets`: https://github.com/broadinstitute/seqr/pull/1663/files#diff-ab443a9f2f134fc3a61c20f3a409e61afa6e604018632336eee9ac36795b2e9bR44